cmd/state-installer/test/integration/installer_int_test.go

```diff
@@ -82,27 +83,15 @@ func (suite *InstallerIntegrationTestSuite) AssertConfig(ts *e2e.Session) {
 	}
 }
 
-func (suite *InstallerIntegrationTestSuite) populateSourceDir(ts *e2e.Session) string {
-	sourceDir := filepath.Join(ts.Dirs.Work, "source")
-	err := fileutils.Mkdir(sourceDir)
-	suite.NoError(err)
-
-	err = fileutils.CopyFile(ts.Exe, filepath.Join(sourceDir, filepath.Base(ts.Exe)))
-	suite.NoError(err)
-	err = os.Remove(ts.Exe)
-	suite.NoError(err)
-
-	err = fileutils.CopyFile(ts.SvcExe, filepath.Join(sourceDir, filepath.Base(ts.SvcExe)))
-	suite.NoError(err)
-	err = os.Remove(ts.SvcExe)
+func (suite *InstallerIntegrationTestSuite) prepareInstaller(ts *e2e.Session) {
+	err := fileutils.CopyFile(ts.InstallerExe, filepath.Join(ts.Dirs.Work, "installer", filepath.Base(ts.InstallerExe)))
```

Comment:
I meant we should update this exact code here:

https://github.com/ActiveState/cli/blob/950bf1083327954d4a1e807c14de37fd26d771f7/internal/testhelpers/e2e/session.go#L138-L138

Because the behaviour is part of the installer, if we don't address it there it'll keep biting us.